stbt-run

```diff
@@ -1,21 +1,20 @@
-#!/usr/bin/env python
+#!/usr/bin/python
```

Comment:
Why removing `/usr/bin/env`? I think it was made that way explicitly for virtualenv support.

Reply:
A mistake, fixed.